changelog.d/graphql_endpoint_toggle.enhancement.md

```diff
@@ -0,0 +1,3 @@
+Added a boolean `graphql` field to the api configuration to allow disabling the graphql endpoint.
+
+Note that the `playground` endpoint will now only be enabled if the `graphql` endpoint is also enabled.
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏I'm not super familiar with this area but I'm wondering if this would be considered a breaking change? It it is that would probably be better to separate out into a second fragment addition.

Reply:
I don't think it's breaking as the `playground` field defaults to true, as it was before.

Reply:
^

Reply:
roger that 👍